ingest/cli_parser.py

```diff
@@ -331,6 +331,20 @@ def create_parser():
         "--gene-file", help="Path to .genes.tsv file"
     )
 
+    # h5ad subparsers
+    parser_h5ad = subparsers.add_parser(
+        "ingest_h5ad", help="Indicates that h5ad file is being ingested"
+    )
+
+    parser_h5ad.add_argument(
+        "--ingest-h5ad",
+        required=True,
+        action="store_true",
+        help="Indicates that ingest of h5ad file should be invoked",
+    )
```

Comment:
This strikes me as unnecessary.  If it's necessary, could you refine `help` to briefly (perhaps further) indicate the purpose of this flag as distinct from the subparser argument (`ingest_h5ad`).  If it's unnecessary, could you remove it?

Reply:
Historically, it seemed necessary for the "run_ingest" function to work (Eno had indicated that there wasn't a good way to find out what subparser was invoked without passing a subparser-unique argument).  I might have found an argparse workaround to that difficulty, I'll do some testing and see.

Reply:
Will look into this simplification in my next batch of coding for Ingest Pipeline (We don't have good test coverage of the CLI interface within this repo so the testing required is more extensive than I have time for in this sprint).